### DISCUSSION TITLE
GetTableName() for different db schema

### DISCUSSION THREAD

**Discussion by c-nv-s** [doytowin/goooqo#16] [General]
in Postgres, databases can be "namespaced" using different schemas, with the default schema being "public".
so by default on a db connection the GetTableName() which if for example returned ``t_users`` would be referring to the table at ``public.t_users``.

However it would be useful to be able to reference different schemas e.g:
public.t_users
org1.t_users
org2.t_users
financeteam.t_users
etc

**Comment by f0rb** [comment]
Currently, the returned value of `GetTableName()` is simply spliced ​​into the table name position in the CRUD statements.
So I think you can define the schema with the table name together:
```go
func (e TestEntity) GetTableName() string {
	return "org1.t_user"
}
```

I'm not sure if you want to use one struct to reference user tables in different schemas.
